templates/books/book_form.html

```diff
@@ -40,26 +40,22 @@
 <div class="span-3"><label for="id_book_file">File:</label></div>
 <div class="span-4">{{ form.book_file }}</div>
 <hr class="space">
-<div class="span-3"><label for="id_a_title">atom:title:</label></div>
+<div class="span-3"><label for="id_a_title">Title:</label></div>
 <div class="span-4">{{ form.a_title }}</div>
 <hr class="space">
-<div class="span-3"><label for="id_a_author">atom:author:</label></div>
+<div class="span-3"><label for="id_a_author">Author:</label></div>
 <div class="span-4">{{ form.a_author }}</div>
 <hr class="space">
-<hr class="space">
-<div class="span-3"><label for="id_a_status">atom:status:</label></div>
+<div class="span-3"><label for="id_a_status">Status:</label></div>
 <div class="span-4">{{ form.a_status }}</div>
 <hr class="space">
 <div class="span-3"><label for="id_tags">Tags:</label></div>
 <div class="span-4 last">{{ form.tags }}</div>
 <hr class="space">
-<div class="span-3"><label for="id_downloads">Downloads:</label></div>
-<div class="span-4 last">{{ form.downloads }}</div>
-<hr class="space">
-<div class="span-3"><label for="id_a_summary">atom:summary:</label></div>
+<div class="span-3"><label for="id_a_summary">Summary:</label></div>
 <div class="span-4"></div><div class="span-8">{{ form.a_summary  }}</textarea></div>
 <hr class="space">
-<div class="span-3"><label for="id_dc_language">dc:language:</label></div>
+<div class="span-3"><label for="id_dc_language">Language:</label></div>
```

Comment:
Is the namespace of the tag coming from Dublincore relevant to the use of this page for 99% of our users? Almost certainly not.